src/authorization-request/URI.ts

```diff
@@ -151,15 +152,13 @@ export class URI implements AuthorizationRequestURI {
     }
 
     const isJwt = typeof authorizationRequestPayload === 'string';
-    const requestObjectJwt = requestObject
-      ? await requestObject.toJwt()
-      : typeof authorizationRequestPayload === 'string'
```

Comment:
I changed this as it was before. However, removing it should have been okay since the AuthorizationRequestPayload cannot be a string based on its type.